go.mod

```diff
@@ -64,6 +64,11 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/hupe1980/go-huggingface v0.0.15 // indirect
+	github.com/sashabaranov/go-openai v1.27.0 // indirect
+)
```

Comment:
You can merge all `require` and then execute `go mod tidy` formatting, which normally only consists of two parts: direct and indirect.